CHANGELOG.md

```diff
@@ -58,22 +89,6 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 -
```

Comment:
nit: I think we can remove the empty sections. We've done that with past releases.